Woof/Woof/Model/LocalStorage/LocalStorage.swift

```diff
@@ -0,0 +1,44 @@
+import Foundation
+
+/// A persistent storage designed for managing a collection of values.
```

Comment:
nit:

Should we describe the fact our storage saves the raw data (byte buffers)? Should we update API then (parameter names and documentation)?

Reply:
Added description into doc comments for data type